src/SurfaceFluxes.jl

```diff
@@ -571,16 +585,18 @@ function heat_exchange_coefficient(
 )
     thermo_params = SFP.thermodynamics_params(param_set)
     FT = eltype(L_MO)
-    Δθ = TD.dry_pottemp(thermo_params, ts_in(sc)) - TD.dry_pottemp(thermo_params, ts_sfc(sc))
+    transport = UF.HeatTransport()
     ustar = compute_ustar(param_set, L_MO, sc, uft, scheme)
-    θstar = Δθ * compute_physical_scale_coeff(param_set, sc, L_MO, UF.HeatTransport(), uft, scheme)
-    ρ_in = TD.air_density(thermo_params, ts_in(sc))
-    ρ_sfc = TD.air_density(thermo_params, ts_sfc(sc))
-    Δρ = ρ_in - ρ_sfc
-    Ch = if Δθ ≈ FT(0) || windspeed(sc) ≈ FT(0) || Δρ ≈ FT(0)
-        FT(0)
+    DSEᵥ_sfc = compute_DSEᵥ(param_set, sc, ts_sfc(sc), z_sfc(sc))
+    DSEᵥ_in = compute_DSEᵥ(param_set, sc, ts_in(sc), z_in(sc))
+    ΔDSEᵥ = DSEᵥ_in - DSEᵥ_sfc
+    ϕ_heat = compute_physical_scale_coeff(param_set, sc, L_MO, transport, uft, scheme)
+    z0_b = z0(sc, UF.HeatTransport())
+    z0_m = z0(sc, UF.MomentumTransport())
+    Ch = if isapprox(ΔDSEᵥ, FT(0); atol=FT(1e-3), rtol=0)
+        Ch = κ^2 / (log(Δz(sc) / z0_b) * log(Δz(sc) / z0_m))
```

Comment:
is this kappa defined anywhere? I imported this branch into ClimaAtmos and ran with it. It pointed me to an errors saying 
```
ERROR: LoadError: UndefVarError: κ not defined
Stacktrace:
  [1] heat_exchange_coefficient(param_set::SurfaceFluxes.Parameters.SurfaceFluxesParameters{Float64, SurfaceFluxes.UniversalFunctions.BusingerParams{Float64}, Thermodynamics.Parameters.ThermodynamicsParameters{Float64}}, L_MO::Float64, sc::SurfaceFluxes.ValuesOnly{Float64, SurfaceFluxes.InteriorValues{Float64, Tuple{Float64, Float64}, Thermodynamics.PhaseEquil{Float64}}, SurfaceFluxes.SurfaceValues{Float64, Tuple{Float64, Float64}, Thermodynamics.PhaseEquil{Float64}}}, uft::SurfaceFluxes.UniversalFunctions.BusingerType, scheme::SurfaceFluxes.FVScheme)
    @ SurfaceFluxes ~/.julia/packages/SurfaceFluxes/E6JnW/src/SurfaceFluxes.jl:597
```

Reply:
`von_karman_const(param_set)` is the function required to access it;

Reply:
@jiahe23 please see the added tests + fix for the above parameter issue in the latest commits.